Books/render-line-item.js

```diff
@@ -0,0 +1,31 @@
+import { calcLineTotal } from './common/utils.js';
+import { toUSD } from './common/utils.js';
+
+function renderLineItem(lineItem, book) {
+    const tr = document.createElement('tr');
+
+    const nameCell = document.createElement('td');
+    nameCell.className = 'align-left';
+    nameCell.textContent = book.name;
+    tr.appendChild(quantityCell);
+
+    const quantityCell = document.createElement('td');
+    quantityCell.textContent = lineItem.quantity;
+    tr.appendChild(priceCell);
+
+    const priceCell = document.createElement('td');
+    priceCell.textContent = toUSD(book.price);
+    tr.appendChild(priceCell);
+    
+
+    const totalCell = document.createElement('td');
+    totalCell.className = 'line-item-total';
```

Comment:
I don't see any `line-item-total` class in your CSS file. Make sure to get rid of old code if you're no longer going to style it with this class.